web/src/engine/websites/HqNow.ts

```diff
@@ -0,0 +1,173 @@
+import { Tags } from '../Tags';
+import icon from './HqNow.webp';
+import { Chapter, DecoratableMangaScraper,Manga,Page,type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchGraphQL, FetchRequest } from '../FetchProvider';
+
+type APIMangas = {
+    getHqsByNameStartingLetter: APIManga[],
+};
+
+type APISingleManga = {
+    getHqsById: APIManga[],
+};
+
+type APIManga = {
+    id: number,
+    name: string,
+    capitulos: APIChapter[]
+};
+
+type APIChapter = {
+    name: string,
+    id: number,
+    number: number
+};
+
+type APIPages = {
+    getChapterById: {
+        pictures: { pictureUrl: string }[]
+    }
+};
+
+const apiUrl = 'https://admin.hq-now.com';
+
+@Common.ImageAjax()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('hqnow', `Hq Now`, 'https://www.hq-now.com', Tags.Language.Spanish, Tags.Media.Comic, Tags.Source.Aggregator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override ValidateMangaURL(url: string): boolean {
+        return /https?:\/\/www\.hq-now\.com\/hq\/([\d]+)/.test(url);
+    }
+    public override async FetchManga(provider: MangaPlugin, url: string): Promise<Manga> {
+        const id = parseInt(new URL(url).pathname.match(/\/hq\/([\d]+)/)[1]);
+        const gql = {
```

Comment:
The other implementations are using 3 separate variables, this is a composed object.
It would be nice trying to keep consistency if possible.